static/app/components/profiling/flamegraph/aggregateFlamegraph.tsx

```diff
@@ -207,6 +207,10 @@ export function AggregateFlamegraph(): ReactElement {
         newView.setConfigView(position.view, {width: {min: 0}});
       }
 
+      // Set to 3/4 of the view up, magic number... Would be best to comput some weighted visual score
+      // based on the number of frames and the depth of the frames, but lets see if we can make it work
+      // with this for now
+      newView.setConfigView(newView.configView.withY(newView.configView.height * 0.75));
```

Comment:
this would break all the highlights wouldnt it? since we set the config view to show the highlighted frames then this will pans to a fixed point which would hide the highlighted frames.

i think you want this inside an else block to not affect the other codepaths

Reply:
I dont think we ever use this on aggregate flamegraphs btw, I can just remove all the code associated with it.